samples/startup-script/package.json

```diff
@@ -4,15 +4,19 @@
   "description": "Start a Google Compute Engine and run the startup script.",
   "main": "index.js",
   "dependencies": {
-    "@google-cloud/compute": "0.10.0"
+    "@google-cloud/compute": "0.10.0",
+    "axios": "^0.18.0"
```

Comment:
Sorry to jump in late. Why are we using `axios` instead of `node-fetch`? Axios is not maintained anymore.

Reply:
Changes committed as suggested.